### DISCUSSION TITLE
Examples using Pluto

### DISCUSSION THREAD

**Discussion by ProfLeao** [ClapeyronThermo/Clapeyron.jl#253] [Ideas]
Hi, I'm think to rewrite the examples (ipynb files) using Pluto. Is there any interest? Would they be merged?

**Comment by longemen3000** [comment]
Hi,

if you want to put up the work, I will be delighted to add the resulting pluto notebooks!. We are aware of Pluto.jl, there is previous work on creating examples with Clapeyron and Pluto (see https://github.com/ClapeyronThermo/introduction-to-computational-thermodynamics), but rewriting the current notebooks to pluto is a great undertaking.

Keep in mind that the current notebooks are really outdated, so while we try to maintain compatibility,there could be places where the results aren't the same. (check https://github.com/ClapeyronThermo/Clapeyron.jl/issues/173)

**Reply by ProfLeao** [reply]
All right, let's do it!!!

**Reply by ProfLeao** [reply]
The work will be done here: https://github.com/ProfLeao/Clapeyron.jl

**Comment by pw0908** [comment]
This might be of interest to you: https://github.com/ClapeyronThermo/Clapeyron.jl/tree/docs-update/examples.

I've been updating the Clapeyron docs and notebooks on a separate branch just so that they are both up-to-date and are more-useful in terms of learning how to use Clapeyron.

**Reply by ProfLeao** [reply]
I'm working on these files now!
